src/eei.cpp

```diff
@@ -740,11 +776,11 @@ string toHex(evmc_uint256be const& value) {
 
       switch (call_result.status_code) {
       case EVMC_SUCCESS:
-        return Literal(uint32_t(0));
+        return Literal(uint32_t(1));
```

Comment:
The spec currently says 0 for success, 1 for failure and 2 for revert (since it is a POSIX style enum and not a boolean anymore).

Reply:
ok, it makes sense to conver this in evm2wasm.  since its EVM that uses 1 for success and 0 for failure (opposite of the POSIX standard)

Reply:
I believe you can do `uint32_t(EVMC_FAILURE)`.

Reply:
How is it related to `EVMC` error codes?